snippets/csharp/System.Text.RegularExpressions/Regex/Overview/regex_example1.cs

```diff
@@ -23,9 +23,12 @@
       // Add the decimal separator.
       pattern += Regex.Escape(nfi.CurrencyDecimalSeparator) + "?";
       // Add the fractional digits.
-      pattern += @"\d{";
+      pattern += @"(\d{";
```

Comment:
NIT: Indenting is slightly off on these two lines

Reply:
Was on the Console lines, not this marked line. Fixed.